src/components/InvestWithdraw.tsx

```diff
@@ -56,7 +56,7 @@ export const InvestWithdraw = () => {
 
 	// TODO: fix missing prop
 	return (
-		<Tabs>
+		<Tabs tabBarExtraContent={undefined}>
```

Comment:
`yarn build` complained when doing @ts-ignore here, not sure if this has any side effects doing it like this